src/test/java/org/opensearch/security/support/Base64HelperTest.java

```diff
@@ -32,17 +32,14 @@
 
 import com.google.common.io.BaseEncoding;
 
-import static com.amazon.opendistroforelasticsearch.security.support.Base64Helper.deserializeObject;
-import static com.amazon.opendistroforelasticsearch.security.support.Base64Helper.serializeObject;
-
 public class Base64HelperTest {
 
     private static final class NotSafeSerializable implements Serializable {
         private static final long serialVersionUID = 5135559266828470092L;
     }
 
     private static Serializable ds(Serializable s) {
-        return deserializeObject(serializeObject(s));
+        return Base64Helper.deserializeObject(Base64Helper.serializeObject(s));
```

Comment:
Not sure if the change is necessary.

Reply:
Same issue with other lines in this file

Reply:
Reverted